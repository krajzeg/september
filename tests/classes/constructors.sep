Person := Class.new("Person")
Person::"<constructor>" = |first_name, last_name| {
	this:first_name = first_name
	this:last_name = last_name
}

john_doe := Person("John", "Doe")
print("This should be John Doe:", john_doe.first_name, john_doe.last_name)
