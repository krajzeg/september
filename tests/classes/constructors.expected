This should be John Doe: John Doe
